src/style.css

```
@@ -1 +1,89 @@
-/* Styles go here */
+body {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  --blueLight-color: #00acdc;
+  --white-color: #fff;
+  --black-color: #000;
+  --size-logo: 40px;
+}
+
+ul {
+  padding: 0;
+}
+
+.menu__head {
+  display: flex;
+  align-content: center;
+  background: var(--white-color);
+  background-color: var(--white-color);
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+  font-size: 12px;
+  line-height: 14px;
+  font-weight: 500;
+  text-transform: uppercase;
+}
+
+.menu_logo {
+  height: var(--size-logo);
+  width: var(--size-logo);
+  margin: 10px auto 10px 0;
+}
+
+.nav {
+  display: flex;
+}
+
+.nav .nav_list {
+  display: flex;
+  margin: 0;
+  list-style-type: none;
+}
+
+.nav_item {
+  margin-right: 20px;
+}
+
+.nav_item:nth-child(4) {
+  flex-shrink: 0;
+}
+
+.nav_item .nav_link {
+  display: block;
+  position: relative;
+  padding-top: 23px;
+  padding-bottom: 23px;
+  text-decoration: none;
+  margin: auto;
+  color: var(--black-color);
+}
+
+.nav_item:last-child {
+  margin-right: 0;
+}
+
+.nav .nav_list .nav_item .nav_link:hover {
+  color: var(--blueLight-color);
+}
+
+.is-active {
+  color: var(--blueLight-color);
+}
+
+.is-active::after {
+  display: inline-block;
+  position: absolute;
+  content: "";
```

```suggestion
  content: "";
  display: inline-block;
  position: absolute;
```

start  `:after` from `content` property

---

done